app/main.py

```diff
@@ -1 +1,68 @@
-#  write your code here
+class Animal:
+    list_animals = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+        Animal.list_animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            self.appetite = 0
+            return self.appetite
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = False
+        return self.appetite
+
+
+class Cat(Animal):
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
```

Comment:
Do not add `appetite` in the `__init__` for `Cat` and `Dog` classes, remove them as mentioned in the task description and pass required value directly to the `super().__init__`